blocks/cards/cards.css

```diff
@@ -347,12 +381,26 @@ main .nolink .cards > ul > li > .cards-card-body > h6 {
   }
 
   main .block.cards.machine-parts > ul > li {
-    width: inherit;
-    flex: 1;
+    /*width: inherit;
+    flex: 1;*/
   }
 
-  main .block.cards.machine-parts > ul > li .cards-card-image {
-    height: 17.1875rem;
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
This seems very "magic number" to me, could we maybe change this breakpoint to something already used in the project?

Reply:
Done